flink-yarn/src/main/java/org/apache/flink/yarn/entrypoint/YarnEntrypointUtils.java

```diff
@@ -46,6 +49,17 @@
  */
 public class YarnEntrypointUtils {
 
+	public static SecurityContext installSecurityContext(
```

Comment:
This seems to not be used anywhere.

Reply:
yes. this was suppose to be part of the rename PR. I will remove this from current PR.